src/components/common/Stepper.vue

```diff
@@ -12,8 +12,7 @@
         <div class="step__line" />
         <v-btn
           :id="step.id"
-          outlined
-          fab
```

Comment:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/beefec5d-caf0-41ba-b1ac-fa59f362e3ba)

Reply:
You fixed it. Resolving.